src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace administrator\components\com_users;
```

Comment:
Usually we do not namespace the cests, do we?

Reply:
Yeah. You are right.